templates/base.html

```diff
@@ -1,7 +1,7 @@
 {% load analytics_tags %}
 
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" {% block html_tag_attrs %}{% endblock %}>
```

Comment:
Why is this needed?

Reply:
Can take this out now

Reply:
I take that back, this is necessary to get the document height to 100% to make sure the iframe is sized correctly.